android/app/build.gradle

```diff
@@ -103,7 +103,10 @@ android {
         applicationId "com.pridelondonapp"
         minSdkVersion 16
         targetSdkVersion 22
-        versionName "1.0.3"
+        versionName "1.0.4"
+        Properties props = new Properties()
+        props.load(new FileInputStream(file("../.gradle/gradle.properties")))
+        versionCode props['versionCode'].toInteger()
```

Comment:
This is the build number. Hasn't been needed for HockeyApp but Fabric only adds a new download if the build increments